util/opts.rb

```diff
@@ -15,25 +38,4 @@ def self.parse_flag(h, f)
       h[name.to_sym] = val.size == 1 ? val.first : val
     end
   end
-
-  def self.parse(args = ARGV)
-    config = OpenStruct.new
-    if args.size > 0
-      config = OpenStruct.new(**args.reduce(Hash.new) do |h, v|
-                                  if v.start_with?(FLAG_PREFIX)
-                                    parse_flag(h, v)
-                                  else
-                                    parse_command(h, v)
-                                  end
-                                  h
-                                end)
-    end
-    config
-  end
-
-  PARSED = parse()
-
-  def self.method_missing(method, *args)
-    PARSED.send(method, *args)
```

Comment:
this is interesting there's some good ways to apply this in an object oriented pattern. using `send` is a bit of an anti pattern and should only be used in cases where it's absolutely necessary.